DigitalLearningSolutions.Web.AutomatedUiTests/AccessibilityTests/BasicAuthenticatedAccessibilityTests.cs

```diff
@@ -13,7 +13,6 @@ public class BasicAuthenticatedAccessibilityTests : AccessibilityTestsBase,
         [Theory]
         [InlineData("/MyAccount", "My account")]
         [InlineData("/MyAccount/EditDetails", "Edit details")]
-        [InlineData("/Signposting/LaunchLearningResource/3", "View resource \"Test image resource\"")]

```

Comment:
Removed this test because there are no LearningResourceReferences in the remote test db, don't know if it's worth adding one in there?

Reply:
Weirdly it kept failing for me the other day, but I've just added it back now and it works fine. Not sure what's up with that but I suppose all's well

Reply:
Odd. Guess we'll keep an eye out for it if it becomes problematic again.